LinearApproach/LinearApproachMultiCCA.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Since you've now added the code to the repo, please update this notebook so it no longer calls local functions, and instead demonstrates how to use <code>lp_pmd</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/sparsecca/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>this will only be one call tho: w= lp_pmd(dataset, K) is this what you mean?</p>